adoc/Private-Registry-Administration.adoc

```diff
@@ -743,3 +743,206 @@ suse-registry-harbor-registry-65854df7bc-mrfnj     2/2     Running   0
 suse-registry-harbor-trivy-0                       1/1     Running   1          24h
 suse-registry-harbor-trivy-1                       1/1     Running   0          24h
 ----
+
+=== Backup and Restore
+
+As an application that is deployed on kubernetes, backup and restore for {spr} can be performed by link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[Velero]. Velero is an application that performs backup/restore, disaster recovery, and migrate Kubernetes cluster resources and persistent volumes.
+
+Velero is able to fully cover the backup/restore for {spr} when it is deployed using internal services, such as database and redis. However, as Velero only covers kubernetes resources, when {spr} is deployed using external service providers, such as managed PostgreSQL database or object storage (Amazon S3/Azure Blob Storage), additional actions should be performed to ensure the backup of data external to the kubernetes cluster.
+
+In general, the backup and restore process for {spr} can be split in two parts:
+
+* Backup/restore of kubernetes resources (performed by Velero):
+** Namespace
+** Deployment
+** ReplicaSet
+** StatefulSet
+** Endpoint
+** Service
+** Ingress
+** ConfigMap
+** Secret
+** PersistentVolumeClaim
+** PersistentVolume
+** Pod
+** Other CRDs (e.g. Certificate when using cert-manager or RedisFailover when using redis-operator)
+* Backup/restore of external services:
+** Database
+** Redis
+** Object Storage
+
+==== Backup/restore of kubernetes resources
+
+The following steps describe how to perform the backup and restore of kubernetes resources using Velero.
+
+===== Backup
+
+. Install Velero as described in its link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[documentation]
+
+. (optional) If using a volume provider does not support volume snapshot or volume snapshot API, or does not have Velero supported storage plugin, Velero leverages link:https://restic.net/[restic] as an agnostic solution to backup and restore this sort of persistent volumes. To perform a restic backup of the persistent volumes Velero requires to add an specific annotation (`backup.velero.io/backup-volumes=<VOLUME_NAME_1>,<VOLUME_NAME_2>,...`) to the pods which have the volume(s) mounted. This can be achieved by adding the following entries to the {spr} helm chart `harbor-values.yaml` and performing helm upgrade to apply the annotations:
```

Comment:
```suggestion
. (optional) If you are using a volume provider that does not support the volume snapshot, the volume snapshot API, or does not have Velero supported storage plugin, Velero leverages link:https://restic.net/[restic] as an agnostic solution to backup and restore persistent volumes. To perform a restic backup of the persistent volumes, Velero requires the addition of a specific annotation (`backup.velero.io/backup-volumes=<VOLUME_NAME_1>,<VOLUME_NAME_2>,...`) to the pods that have the volumes mounted. This can be achieved by adding the following entries to the {spr} helm chart `harbor-values.yaml` and performing `helm upgrade` to apply the annotations:

Reply:
done